### ISSUE TITLE
tabBar.barStyle and navigationBar.barStyle stopped working with Swift 4 

### ISSUE THREAD

**Issue by n3d1117** [jiecao-fm/SwiftTheme#41]
Hello, I recently updated to Swift 4 and noticed that this stopped working:
```
let navigationBar = UINavigationBar.appearance()
navigationBar.theme_barStyle = [.default, .black]

let tabBar = UITabBar.appearance()
tabBar.theme_barStyle = [.default, .black]
```
All I see is the default style that doesn't change when switching theme. Tested on both real device and simulator. However setting `tabBar.barStyle = .black` manually seems to work just fine. Any help?

**Comment by shannonchou** [comment]
I verified that there really is the bug. We may fix it several days later. Thanks.

**Comment by trusk89** [comment]
Is there any update on this?

**Comment by wxxsw** [comment]
@n3d1117 @trusk89 

I fix this problem in the new version 0.4.0, this new version uses Swift 4.

In addition, if you are still Swift 3 project, run `pod install` or `pod update` will generate the corresponding Swift 3 Pod configuration, this can be solved by:

```ruby
target 'YourTarget' do
  use_frameworks!

  pod 'SwiftTheme'
  
  post_install do |installer|
    installer.pods_project.targets.each do |target|
      if target.name == 'SwiftTheme'
        target.build_configurations.each do |config|
          config.build_settings['SWIFT_VERSION'] = '4.0'
        end
      end
    end
  end

end
```
> https://github.com/CocoaPods/CocoaPods/issues/6791

If you have any problem, please continue to feedback, thank you!

**Comment by trusk89** [comment]
Can confirm, thank you @wxxsw!
